src/sentry/web/forms/accounts.py

```diff
@@ -553,16 +553,19 @@ def __init__(self, project, user, *args, **kwargs):
         # for notifications to keep their settings
         emails = [e.email for e in user.get_verified_emails()]
         alert_email = UserOption.objects.get_value(user=self.user, project=None, key='alert_email', default=None)
+        if alert_email:
+            alert_email = alert_email.rstrip()
```

Comment:
I don't think we do expect whitespace, as we use the cleaned_data attribute of the form to store alert emails. I was able to replicate the UI issue only with whitespace stored rather than an empty string, though, so I used rstrip as a precaution. I can remove this.